sdk/ml/azure-ai-ml/tests/test_configs/components/additional_includes/docker/DockerFile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:22.04
```

Comment:
This needs to go to a non docker.io source.!

Reply:
switch to mcr.microsoft.com/cbl-mariner/base/core:2.0